src/app/modules/main/wallet_section/activity/entry.nim

```diff
@@ -185,6 +172,7 @@ QtObject:
       return ""
     return self.transaction[].getNFTImageURL()
 
+  # TODO: is this needed for activity history summary? call getOpenseaAssetsByNFTUniqueID when previewing details of the entry
```

Comment:
@dlipicar I plan to follow up with another PR to refactor and remove fields used in details view from this class. At first glance, this is used for details only; can it be removed from here and moved to another file specific for fetching and providing details only when the user visualizes them?

Reply:
Will follow up handling this TODO with work here: https://github.com/status-im/status-desktop/issues/11597